services/auth-service/tsconfig.json

```diff
@@ -99,5 +99,6 @@
     /* Completeness */
     // "skipDefaultLibCheck": true,                      /* Skip type checking .d.ts files that are included with TypeScript. */
     "skipLibCheck": true                                 /* Skip type checking all .d.ts files. */
-  }
+  },
+     "exclude": ["node_modules", "**/*.test.ts"]
```

Comment:
test shouldn't be typechecked by the `tsc`?

Reply:
> the test shouldn't be typechecked by `tsc`? why

Think that's just how it was in the jest example we followed and we didn't bother changing it.